components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -224,548 +76,163 @@ private SessionDataStore() {
-                log.error(e.getMessage(), e);
-                return null;
-            }
-            PreparedStatement preparedStatement = null;
-            ResultSet resultSet = null;
-
-            try {
-                if (StringUtils.isBlank(sqlSelect)) {
-                    if (connection.getMetaData().getDriverName().contains(MYSQL_DATABASE)
-                            || connection.getMetaData().getDriverName().contains(H2_DATABASE)) {
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_MYSQL;
-                    } else if (connection.getMetaData().getDatabaseProductName().contains(DB2_DATABASE)) {
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_DB2SQL;
-                    } else if (connection.getMetaData().getDriverName().contains(MS_SQL_DATABASE)
-                            || connection.getMetaData().getDriverName().contains(MICROSOFT_DATABASE)) {
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_MSSQL;
-                    } else if (connection.getMetaData().getDriverName().contains(POSTGRESQL_DATABASE)) {
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_POSTGRESQL;
-                    } else if (connection.getMetaData().getDriverName().contains(INFORMIX_DATABASE)) {
-                        // Driver name = "IBM Informix JDBC Driver for IBM Informix Dynamic Server"
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_INFORMIX;
-                    } else {
-                        sqlSelect = SQL_DESERIALIZE_OBJECT_ORACLE;
-                    }
-                }
-                preparedStatement = connection.prepareStatement(getSessionStoreDBQuery(sqlSelect, type));
-                preparedStatement.setString(1, key);
-                preparedStatement.setString(2, type);
-                resultSet = preparedStatement.executeQuery();
-                if (resultSet.next()) {
-                    String operation = resultSet.getString(1);
-                    long nanoTime = resultSet.getLong(3);
-                    if ((OPERATION_STORE.equals(operation))) {
-                        return new SessionContextDO(key, type, getBlobObject(resultSet.getBinaryStream(2)), nanoTime);
-                    }
-                }
-            } catch (ClassNotFoundException | IOException | SQLException |
-                    IdentityApplicationManagementException e) {
-                log.error("Error while retrieving session data", e);
-            } finally {
-                IdentityDatabaseUtil.closeAllConnections(connection, resultSet, preparedStatement);
-            }
-            return null;
-        }
-        return null;
+        storeSessionData(key, type, entry, MultitenantConstants.INVALID_TENANT_ID);
     }
 
-    public void persistSessionData(String key, String type, Object entry, long nanoTime, int tenantId) {
+    /**
+     * Storing or updating sessiondata main method by calling persistSessionData method when there is a tenantId
```

Comment:
Enter the line witth a fullstop. Make sure this is followed in every description and comment.